diff --git a/shield.mk b/shield.mk
index 6e8734b..976fda9 100644
--- a/shield.mk
+++ b/shield.mk
@@ -108,3 +108,8 @@ PRODUCT_PACKAGES += \
 # USB
 PRODUCT_PACKAGES += \
     com.android.future.usb.accessory
+
+ADDITIONAL_DEFAULT_PROPERTIES += \
+    ro.adb.secure=0 \
+    ro.secure=0 \
+    ro.debuggable=1
diff --git a/system_prop.mk b/system_prop.mk
index 93bf9dc..587d6c6 100644
--- a/system_prop.mk
+++ b/system_prop.mk
@@ -20,7 +20,7 @@ PRODUCT_PROPERTY_OVERRIDES += \
 
 # USB
 PRODUCT_PROPERTY_OVERRIDES += \
-    persist.sys.usb.config=mtp
+    persist.sys.usb.config=mtp,adb
 
 # WiFi
 PRODUCT_PROPERTY_OVERRIDES += \
